htslib/vcf.h

```diff
@@ -61,16 +61,24 @@ extern "C" {
 
 #define BCF_HT_FLAG 0 // header type
 #define BCF_HT_INT  1
-#define BCF_HT_REAL 2
-#define BCF_HT_STR  3
-#define BCF_HT_LONG (BCF_HT_INT | 0x100) // BCF_HT_INT, but for int64_t values; VCF only!
+#define BCF_HT_REAL 7
```

Comment:
Can you investigate if we add to the original definition instead of replacing them?

Reply:
Yeah, I think we can do that, keep the originals, replace what we use now in the new ones.

Reply:
I reverted changes to HT_REAL and HT_STR but HT_LONG requires more changes on our genomicsdb  side(variant_operations.cc) as it is done according to being HT_LONG is 10.